sentry-samples/sentry-samples-console/src/main/java/io/sentry/samples/console/Main.java

```diff
@@ -1,13 +1,14 @@
 package io.sentry.samples.console;
 
+import java.util.Collections;
```

Comment:
Changes in this file are unrelated to PR. It's just formatting that somehow wasn't taken into account when merging PR with sample console app.